### ISSUE TITLE
clock

### ISSUE THREAD

**Issue by jostephd** [wesnoth/wesnoth#3722]

**Comment by jyrkive** [comment]
Two things.

1. I like the clock. It's useful when playing in fullscreen mode.
2. The reason why I agreed to work on the battery widget is because of the clock: "we're already showing the current time; I agree that showing battery charge as well is a good idea"

**Comment by CelticMinstrel** [comment]
Maybe offer it as a setting in preferences?

**Comment by cooljeanius** [comment]
Yeah the real-world local time display is a useful tool for realizing "oh no I've been playing Wesnoth for too long and really need to stop now"
